src/scripts/wallet/Wallet.ts

```diff
@@ -1,16 +1,14 @@
-/// <reference path="../../declarations/utilities/getArrayOfObservables.d.ts"/>
-
 class Wallet implements Feature {
     name = 'Wallet';
     saveKey = 'wallet';
-    currencies: Array<number>;
+    currencies: Array<KnockoutObservable<number>>;
```

Comment:
I feel much better about these type signatures now, it felt like we were lying before